src/Compilers/VisualBasic/Portable/Symbols/Wrapped/WrappedEventSymbol.vb

```diff
@@ -0,0 +1,120 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    ''' <summary>
+    ''' Represents an event that is based on another event.
+    ''' When inheriting from this class, one shouldn't assume that 
+    ''' the default behavior it has is appropriate for every case.
+    ''' That behavior should be carefully reviewed and derived type
+    ''' should override behavior as appropriate.
+    ''' </summary>
+    Friend MustInherit Class WrappedEventSymbol
+        Inherits EventSymbol
+
+        Protected _underlyingEvent As EventSymbol
```

Comment:
done

---
In reply to: [71573154](https://github.com/dotnet/roslyn/pull/12596#discussion_r71573154) [](ancestors = 71573154)